### ISSUE TITLE
Associate the labels with the form controls for "address of reference document" and  "address of new document"

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/htmldiff-ui#27]
This lack of association causes screen readers not to read the label content when moving to next and previous form field and when tabbing through the page.

Currently:

```
<p>Address of reference document: <input name="doc1" type="url" value="" style="width:100%"></p>
```

Suggested:

```
<label for="address-of-reference-document">Address of reference document:</label><input name="doc1" type="url" value="" style="width:100%" id="address-of-reference-document">
```

And the same for the other input field.
